projects/packages/sync/src/modules/class-callables.php

```diff
@@ -396,13 +396,13 @@ public function set_plugin_action_links() {
 				'edit'       => '',
 			);
 			/** This filter is documented in src/wp-admin/includes/class-wp-plugins-list-table.php */
-			$action_links = apply_filters( 'plugin_action_links', $action_links, $plugin_file, null, 'all' );
+			$action_links = apply_filters( 'plugin_action_links', $action_links, $plugin_file, array(), 'all' );
```

Comment:
Would it be worth trying to provide the actual plugin data, in case something tries to use it?

Looks like we could do that pretty easily by changing the code above from
```php
$plugins = array_keys( Functions::get_plugins() );
foreach ( $plugins as $plugin_file ) {
```
to
```
$plugins = Functions::get_plugins();
foreach ( $plugins as $plugin_file => $plugin_data ) {
```

Reply:
That's a good point. Done in f83ba87c19fa878530aee2ce3bb5f6c158c16abb